packages/web/src/pages/audio-rewards-page/components/WalletsTable.tsx

```diff
@@ -251,7 +253,9 @@ const WalletsTable = ({
           isConfirmRemoving={false}
         />
       )}
-      {errorMessage && <div className={styles.error}>{errorMessage}</div>}
+      {!suppressError && errorMessage ? (
```

Comment:
should we just remove this here, or is it needed for another component usage?

Reply:
Ah I think you're right and we can remove it actually. It's used in the AudioBreakdownModal but don't think we care

Reply:
fixed. good call